docs/config.md

```diff
@@ -230,6 +231,9 @@ names:
 * *loadBalancer* -- Optional.  Specifies a load balancer to use.  It must be an
 object containing keys:
   * *kind* -- One of the supported load balancers.
+  * *probationEnabled* -- Controls whether endpoints are eagerly evicted from
```

Comment:
yeah, matching finagle is nice.  but "enabled" on a boolean is pretty dumb and redundant.  *shrug* I don't feel strongly.